go.mod

```diff
@@ -31,7 +31,7 @@ require (
 	github.com/ipfs/go-graphsync v0.11.0
 	github.com/ipfs/go-ipfs-blockstore v1.1.2
 	github.com/ipfs/go-ipfs-chunker v0.0.5
-	github.com/ipfs/go-ipfs-cmds v0.6.0
+	github.com/ipfs/go-ipfs-cmds v0.6.1-0.20220321213443-7217331bf422
```

Comment:
This should be tagged, Gus could you cut a release when the dependency PR lands, please? (If it's a one-liner feel free to paste it here and I'll take care of it.)

Reply:
Thanks, updating.